guides/api_overview.md

```diff
@@ -311,3 +311,7 @@ You can implement the following callbacks:
 You will receive content share metrics if you registered a metric observer by [9.Receiving metrics (optional)](https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/api_overview.md#9-receiving-metrics-optional).
 
 Content share metrics will be prefixed by `contentShare`.
+
+## 14. Configuring Remote Video Subscriptions
+
+Amazon Chime SDK allows builders to have complete control over the remote videos received by each of their application’s end-users. This can be accomplished using the API [AudioVideoFacade.updateVideoSourceSubscriptions](https://aws.github.io/amazon-chime-sdk-ios/Protocols/AudioVideoControllerFacade.html#/c:@M@AmazonChimeSDK@objc(pl)AudioVideoControllerFacade(im)updateVideoSourceSubscriptionsWithAddedOrUpdated:removed:). See [Configuring Remote Video Subscriptions](/guides/configuring_remote_video_subscriptions.md) for more information.
```

Comment:
AudioVideoControllerFacade.update

Reply:
The links shows AudioVideoControllerFacade. But when builder call the `updateVideoSourceSubscriptions`, they will use `AudioVideoFacade.updateVideoSourceSubscriptions`. Since it implements the AudioVideoFacade as well.